<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2022)/launch/sim_w1.launch" />
    
    <param name="publish_frequency" type="double" value="1.0" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    
    <node pkg="assignment_2_2022" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment_2_2022" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assignment_2_2022" type="bug_as.py" name="bug_action_service"  />
    <node pkg="assignment_2_2022" type="client_publisher.py" name="client_publisher" output="screen" launch-prefix="xterm -hold -e"  />
    <node pkg="assignment_2_2022" type="subscriber.py" name="subscriber" output="screen" launch-prefix="xterm -hold -e"  />
    <node pkg="assignment_2_2022" type="service.py" name="service" output="screen" launch-prefix="xterm -hold -e" />
</launch>
